src/showcase/showcase.html

```diff
@@ -1 +1 @@
-<div id="showcase-container" selection="selection()"></div>
+<div id="showcase-container" selection="selection()" ng-click="save()"></div>
```

Comment:
This was the best way I could think of to force an update of the indicators, any other ideas would be great